tests/sqlparser_mysql.rs

```diff
@@ -4353,3 +4362,41 @@ fn test_create_index_options() {
         "CREATE INDEX idx_name ON t(c1, c2) USING BTREE LOCK = EXCLUSIVE ALGORITHM = DEFAULT",
     );
 }
+
+#[test]
+fn test_insert_into_value() {
+    let sql = r"INSERT INTO t (id, name) VALUE ('AAA', 'BBB')";
+    match mysql_and_generic().verified_stmt(sql) {
+        Statement::Insert(Insert { source, .. }) => {
+            assert_eq!(
+                Some(Box::new(Query {
+                    with: None,
+                    body: Box::new(SetExpr::Values(Values {
+                        keyword: ValuesKeyword::Value,
+                        explicit_row: false,
+                        rows: vec![vec![
+                            Expr::Value(ValueWithSpan {
+                                value: Value::SingleQuotedString("AAA".to_string()),
+                                span: Span::empty(),
+                            }),
+                            Expr::Value(ValueWithSpan {
+                                value: Value::SingleQuotedString("BBB".to_string()),
+                                span: Span::empty(),
+                            }),
+                        ]],
+                    })),
+                    order_by: None,
+                    limit_clause: None,
+                    fetch: None,
+                    locks: vec![],
+                    for_clause: None,
+                    settings: None,
+                    format_clause: None,
+                    pipe_operators: vec![],
+                })),
+                source
+            );
+        }
+        _ => unreachable!(),
+    }
```

Comment:
```suggestion
    mysql_and_generic().verified_stmt(sql);
```
I think this format should suffice since we cover the AST assertions in other tests

Reply:
Also instead of adding a oneoff test here, can we add a test case to the existing test function [here](https://github.com/SatoriCyber/datafusion-sqlparser-rs/blob/0ebcaae646d4f52c23462b9715fd92f9fd335b7b/tests/sqlparser_common.rs#L99)?